test/parse-args.js

```diff
@@ -21,6 +21,17 @@ describe('parse-args', () => {
       const argv = buildYargs().parse(hideInstrumenteeArgs())
       const instrumenteeArgs = hideInstrumenterArgs(argv)
       instrumenteeArgs.should.eql(['my-app', '--help'])
+      argv.tempDirectory.endsWith('/coverage/tmp').should.be.equal(true)
+    })
+  })
+
+  describe('with NODE_V8_COVERAGE already set', () => {
+    it('should not override it', () => {
+      process.env.NODE_V8_COVERAGE = './coverage/tmp_'
+      process.argv = ['node', 'c8', '--foo=99', 'my-app', '--help']
+      const argv = buildYargs().parse(hideInstrumenteeArgs())
+      argv.tempDirectory.endsWith('/coverage/tmp_').should.be.equal(true)
+      process.env.NODE_V8_COVERAGE = ''
```

Comment:
I assume the original value is not `''`, as this test is run with `c8`.

Reply:
I can `delete` but that would create a side effect like setting it to `''`. I can use `beforeEach` and store the original value. I am only doing it that particular test now.